libnmstate/nm/sriov.py

```diff
@@ -72,8 +162,36 @@ def get_info(device):
     )
 
     if sriov_setting:
-        info[Ethernet.SRIOV_SUBTREE] = {
-            Ethernet.SRIOV.TOTAL_VFS: sriov_setting.props.total_vfs
-        }
+        sriov_config = {}
+        ifname = device.get_iface()
+        with open(f"/sys/class/net/{ifname}/device/sriov_numvfs") as f:
```

Comment:
As we promised in API document: https://nmstate.github.io/devel/api.html#error-handling

A `try exception` should be used here to wrap unexpected error  to `NmstateInternalError`

Reply:
Done, the commit have been moved to #670